src/Microsoft.AspNetCore.Hosting/project.json

```diff
@@ -50,7 +52,7 @@
         }
       }
     },
-    "netstandard1.3": {
+    "netstandard1.5": {
```

Comment:
If previously I wanted a Kestrel upgrade stream/filter support to 4.6+ & core I'd just need to use `netstandard1.3`; however now will need to use both `net46` and `netstandard1.5` as there isn't an upgrade path between them